ligo/token/fa2.mligo

```diff
@@ -0,0 +1,108 @@
+// SPDX-FileCopyrightText: 2021 Arthur Breitman
```

Comment:
The makefile is not picking up changes made in files under `ligo/token/**`.

For example, if I compile the project using `make all`, and then edit something in ` ligo/token/fa2.mligo` and run `make all` again, I'll see this:

```
$ make all
make: Nothing to be done for 'all'.
```

I think we need to change line 44 from:
```
$(OUT)/segmented_cfmm_%.tz: ligo/**
```

To:
```
$(OUT)/segmented_cfmm_%.tz: $(shell find ligo -name '*.mligo')
```

Reply:
Nice, didn't notice that it was not working properly. I've updated this now.